lib/cli.js

```diff
@@ -382,11 +382,22 @@ export default async function loadCli() { // eslint-disable-line complexity
 	}
 
 	let parallelRuns = null;
+	let parallelRunsComparator = null;
 	if (isCi && ciParallelVars) {
 		const {index: currentIndex, total: totalRuns} = ciParallelVars;
 		parallelRuns = {currentIndex, totalRuns};
 	}
 
+	if (parallelRuns) {
+		if (Reflect.has(conf, 'ciParallelRunsComparator') && typeof conf.ciParallelRunsComparator !== 'function') {
+			exit('ciParallelRunsComparator must be a comparator function.');
+		}
+
+		const defaultComparator = (a, b) => a.localeCompare(b, [], {numeric: true});
+		// The function needs to be serializable to support worker threads
```

Comment:
It doesn't, `lib/api.js` runs in this process and the function is not needed in the worker threads. If there is a serialization error it needs to be stripped out instead.

Reply:
Ah, thanks for the tip! Fixed in https://github.com/avajs/ava/pull/2968/commits/07c3eff3a3c6c6857992b5c47942ebef8a8e034c